assets/src/scss/new-identity/style.scss

```diff
@@ -9,4 +9,22 @@
   --block-columns--step-number--background: var(--gp-green-100);
   --block-columns--step-number--color: var(--grey-900);
   --block-columns--card-header--background: var(--p4-dark-green-800);
+  --link--focus--outline: 2px solid #99c3ff;
+  --top-page-tags--tag-item--background: var(--gp-green-100);
+  --post-tag-button--font-weight: 700;
+  --post-tag-button--border-color: var(--gp-green-100);
+  --post-tag-button--hover--color: var(--grey-900);
+  --post-tag-button--background-color: var(--gp-green-100);
+  --post-tag-button--hover--background-color: var(--gp-green-200);
+  --post-tag-button--active--background-color: var(--gp-green-200);
+  --post-tag-button--visited--color: var(--grey-900);
+  --post-tag-button--active--color: var(--grey-900);
+}
+
+.post-tag-button {
+  color: var(--grey-900) !important;
```

Comment:
We do have a `--post-tag-button--color` variable, why aren't you using it instead of this line? 🤔